earth_enterprise/src/common/serverdb/serverdbReader.cpp

```diff
@@ -396,7 +442,12 @@ const MimeType ServerdbReader::GetData(
     } else if (request == "Json") {
       std::string json_variable_name = arg_map["var"];
       // CheckJavascriptName(json_variable_name);
-      GetJson(json_variable_name, language, region, buf);
+      if (arg_map["v"] == "2") {
+        GetJson(json_variable_name, language, region, buf);
+      } else {
```

Comment:
if one pass a different version from 1 or 2 (valid options) we still want to return the v 1 result ? Let's add validation for the passed argument.

Reply:
`v` parameter is optional and if not include we want default behavior to use original behavior (`v=1` behavior).  We could raise an exception if the `v` parameter present and not a value of `1` or `2`.  But if `v` is not present we should assume `v=1`.